composer.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "dmitryd/typo3-realurl",
+    "name": "svendeka/typo3-realurl",
```

Comment:
I'm relatively sure this shouldn't be changed ;)

Reply:
yeah, your right. testing purposes. will be changed back. thx!